pkg/controller/builder_test.go

```diff
@@ -0,0 +1,9 @@
+package controller
+
+import (
+	"testing"
+)
+
+func TestControllerNameForObject(t *testing.T) {
+	// TODO: Add test cases.
+}
```

Comment:
Marked the PR as draft, mainly because I wanted to check to see if this is what you had in mind in terms of the scope of the testing in this comment: https://github.com/openshift/cluster-olm-operator/pull/25#discussion_r1263808519

Was it [this the function](https://github.com/openshift/cluster-olm-operator/pull/28/files#diff-a3a53f8da96ba212631a3dcbaefe688da5491bce9e55daaaf83c20b762bcac4eR131) you were thinking about testing for?

Reply:
Yes, thanks! And more over time too